bpython/curtsies.py

```diff
@@ -122,6 +122,9 @@ def after_suspend():
                 window.__enter__()
                 interrupting_refresh()
 
+            def get_top_usable_line():
+                return window.top_usable_row
+
```

Comment:
Any reason all those functions do not have a default implementation in `Repl` and are overwritten in a derived class taking  `window` and `input_generator`?